spring-boot-modules/spring-boot-properties/src/main/java/com/baeldung/configurationproperties/ImmutableCredentials.java

```diff
@@ -1,10 +1,8 @@
 package com.baeldung.configurationproperties;
 
 import org.springframework.boot.context.properties.ConfigurationProperties;
-import org.springframework.boot.context.properties.ConstructorBinding;
 
 @ConfigurationProperties(prefix = "mail.credentials")
```

Comment:
What's the purpose of introducing these changes, and how are they relevant to current task?